clean/ca/config/monterey_county_district_attorney.py

```diff
@@ -0,0 +1,31 @@
+index_request_headers = {
+    "accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7",
+    "accept-language": "en-US,en;q=0.9,nl;q=0.8,ur;q=0.7,ru;q=0.6",
+    "cache-control": "max-age=0",
+    "if-modified-since": "Tue, 06 Aug 2024 10:41:35 GMT",
```

Comment:
@tarakc02 these are just the headers used in downloading the pages, i have put them in a separate file because i have seen in another scraper it was done this way.